woocommerce/class-sv-wc-helper.php

```diff
@@ -24,6 +24,9 @@
 
 namespace SkyVerge\WooCommerce\PluginFramework\v5_10_6;
 
+use Automattic\WooCommerce\Admin\Features\Navigation\Menu;
+use Automattic\WooCommerce\Admin\Features\Navigation\Screen;
+use Automattic\WooCommerce\Admin\Features\Features;
 use Automattic\WooCommerce\Admin\Loader;
```

Comment:
Good point, @unfulvio. I agree that we should try to prevent ambiguity. Done in https://github.com/skyverge/wc-plugin-framework/pull/533/commits/f3e6cd42482452ef11bf62c75095bf89729f10c2.

Reply:
hey @adrianostanley thanks for doing that - I had to amend the commit because when using fully qualified namespaces they need to begin with the global namespace `\` or they won't work (because PHP will assume they're in the current project namespace), I've fixed it in https://github.com/skyverge/wc-plugin-framework/pull/533/commits/5e844b34f4dd109285635a1cd17be723f2330e6a